nsds_lab_to_nwb/components/neural_data/neural_data_originator.py

```diff
@@ -28,13 +59,23 @@ def make(self, nwb_content, electrode_table_regions):
             data, metadata = self.neural_data_reader.get_data(stream=device_name, dev_conf=dev_conf)
             if data is None:
                 logger.info(f'No data availble for {device_name}. Skipping...')
-            else:
+            else:    
+                
+                #resample data
+                self.hardware_rate = metadata['sample_rate']
+                if self.resample_flag:
+                    data = self.resample(data)                
+                
+                #make description
+                description = self.make_description(device_name)
+                
                 electrode_table_region = electrode_table_regions[device_name]
-                e_series = ElectricalSeries(name=device_name,
+                e_series = ElectricalSeries(name=device_name,                                            
```

Comment:
Trailing spaces (you can probably have VS Code automatically remove them).

Reply:
Indeed. I was using pylint to detect style issues, but it looks like it's pretty lax compared to flake8. Now I set flake8 to detect any style issues and autopep8 to automagically fix them.